tests/test_scripts.py

```diff
@@ -2681,6 +2681,18 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+<<<<<<< HEAD
```

Comment:
On Mon, Feb 23, 2015 at 04:03:14PM -0800, Sherine wrote:
> > @@ -2681,6 +2681,18 @@ def test_trim_low_abund_trimtest_savetable():
> >                  'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
> >  
> >  
> > +<<<<<<< HEAD
> 
> yes, at the end I will remove everything. 

This will prevent your code from running at all, as it is a Python syntax
error.

--t
-- 
C. Titus Brown, ctbrown@ucdavis.edu